src/payment-providers/declarations.d.ts

```diff
@@ -9,58 +9,3 @@ declare module 'jsencrypt' {
     new (): JSEncryptPrototype;
   };
 }
-
-
-interface Symbol {
```

Comment:
Where are these defined elsewhere, or are they just unused?

Reply:
They're defined in the new version of JSEncrypt.